aspnetcore/signalr/version-differences.md

```diff
@@ -0,0 +1,126 @@
+});
+```
+
+### Sticky sessions now required
+
+Because of how scale-out worked in the previous versions of SignalR, clients could reconnect and send messages to any server in the farm. Due to changes to the scale-out model, as well as not supporting reconnects, this is no longer supported. Now, once the client connects to the server it needs to interact with the same server for the duration of the connection.
+
+### Single hub per connection
+
+In ASP.NET Core SignalR, the connection model has been simplified. Connections are made directly to a single hub, rather than a single connection being used to share access to multiple hubs.
+
+### Streaming
+
+SignalR now supports [streaming data](xref:signalr/streaming) from the hub to the client.
+
+### State
+
+The ability to pass arbitrary state between clients and the hub (often called HubState) has been removed, as well as support for progress messages. There is no counterpart of hub proxies at the moment.
+
+## Differences on the client
+
+### TypeScript
+
+The ASP.NET Core version of SignalR is written in [TypeScript](https://www.typescriptlang.org/). You can write in JavaScript or TypeScript when using the [JavaScript client](xref:signalr/javascript-client).
+
+### The JavaScript client is hosted at [npm](https://www.npmjs.com/)
+
+In previous versions, the JavaScript client was obtained through a NuGet package in Visual Studio. For the Core versions, the [@aspnet/signalr npm package](https://www.npmjs.com/package/@aspnet/signalr) contains the JavaScript libraries. This package isn't included in the **ASP.NET Core Web Application** template. Use npm to obtain and install the `@aspnet/signalr` npm package.
+
+```console
+npm init -y
+npm install @aspnet/signalr
+```
+
+### jQuery
+
+The dependency on jQuery has been removed, however projects can still use jQuery.
+
+### JavaScript client method syntax
+
+The JavaScript syntax has changed from the previous version of SignalR. Rather than using the `$connection` object, create a connection using the `HubConnectionBuilder` API.
+
+```javascript
+const connection = new signalR.HubConnectionBuilder()
+    .withUrl("/hub")
+    .build();
+```
+
+Use `connection.on` to specify client methods that the hub can call.
+
+```javascript
+connection.on("ReceiveMessage", (user, message) => {
+    const msg = message.replace(/&/g, "&amp;").replace(/</g, "&lt;").replace(/>/g, "&gt;");
+    const encodedMsg = user + " says " + msg;
+    log(encodedMsg);
+});
+```
+
+After creating the client method, start the hub connection. Chain a `catch` method to log or handle errors.
+
+```javascript
+connection.start().catch(err => console.error(err.toString()));
+```
+
+### Hub proxies
+
+Hub proxies are no longer automatically generated. Instead, the method name is passed into the `invoke` API as a string.
+
+### .NET and other clients
+
+The `Microsoft.AspNetCore.SignalR.Client` NuGet package contains the .NET client libraries for ASP.NET Core SignalR.
```

Comment:
Removing the code fencing from the package name and link to https://www.nuget.org/packages/Microsoft.AspNetCore.SignalR.Client/.